test/parallel/test-crypto-dh.js

```diff
@@ -56,21 +59,29 @@ const secret3 = dh3.computeSecret(key2, 'hex', 'base64');
 
 assert.strictEqual(secret1, secret3);
 
+const wrongBlockLength =
+    new RegExp('^Error: error:0606506D:digital envelope' +
```

Comment:
See discussion [here](https://github.com/nodejs/node/pull/10714#discussion_r95296414). I thought it was 4 spaces (otherwise two would get confused with indentation), but I'm not sure what the consensus is.